src/ui/components/navigation/navigationcomponent.js

```diff
@@ -472,8 +473,22 @@ export default class NavigationComponent extends Component {
     }
 
     // We want to persist the params from the existing URL to the new
-    // URLS we create.
+    // URLS we create, except for facets/filters/pagination
     params.set('tabOrder', this._tabOrder);
+
+    params.delete(StorageKeys.SEARCH_OFFSET);
+
+    let prefixes = this.componentManager.getComponentNamesForComponentTypes([
```

Comment:
This logic is matching the logic detailed in each component. That is, the persistentStorage key started with `this.name`. Should that change in any of these components, it would need to be changed here as well. I did not know if we wanted to invest time in this item for some new system `getStorageKey` so there is one source of truth for these URLs.